examples/CalcQt/CMakeLists.txt

```diff
@@ -9,12 +9,20 @@ if(QT_LIBRARIES)
     add_executable(calcqt src/CalcQt.cpp)
     target_link_libraries(calcqt libcalcqt ${QT_LIBRARIES})
 
+    if(Qt5TEST_FOUND)
+        add_executable(QtTestCalculatorQtSteps features/step_definitions/QtTestCalculatorQtSteps)
+        set_target_properties(QtTestCalculatorQtSteps PROPERTIES AUTOMOC ON)
```

Comment:
Why do you enable AUTOMOC for this? I don't see any moc-able object in your one source file.